1-getting-started-lessons/translations/README.np.md

```diff
@@ -0,0 +1,17 @@
+# वेब विकासको साथ सुरू गर्दै
+
+पाठ्यक्रमको यस खण्डमा, तपाईंलाई व्यावसायिक विकासकर्ता बन्न महत्त्वपूर्ण गैर-प्रोजेक्ट-आधारित अवधारणाहरूसँग परिचय गराइनेछ।
+
+### विषयहरू
+
+1. [प्रोग्रामिङ भाषाहरू र व्यापारका उपकरणहरूको परिचय](1-intro-to-programming-languages/README.md)
+2. [GitHub को परिचय](2-github-basics/README.md)
```

Comment:
hi, can you check the paths of these links, as they might need a deeper relative path

Reply:
> hi, can you check the paths of these links, as they might need a deeper relative path

Done!